client/basic-authorship/src/basic_authorship.rs

```diff
@@ -68,8 +74,17 @@ impl<A, B, C> ProposerFactory<A, B, C> {
 			transaction_pool,
 			metrics: PrometheusMetrics::new(prometheus),
 			_phantom: PhantomData,
+			max_block_size: DEFAULT_MAX_BLOCK_SIZE,
 		}
 	}
+
+	/// Set the maximum block size in bytes.
+	///
+	/// The default value for the maximum block size is:
+	/// [`DEFAULT_MAX_BLOCK_SIZE`].
```

Comment:
Maybe we should document here that there are still network limitations regarding block size (16MB iirc?).

Reply:
The total size of a blocks response is 10 MiB. The actual should be less, since the remote might have requested the header and justification for this block as well.